usaspending_api/awards/models.py

```diff
@@ -144,16 +140,29 @@ def __str__(self):
         return '%s piid: %s fain: %s uri: %s' % (self.get_type_display(), self.piid, self.fain, self.uri)
 
     def __get_latest_submission(self):
```

Comment:
This is nit picky, but what do you think about using a different name for this method, since it's not really getting submissions? In general, would be great to settle on a shared vocabulary around award actions/transactions/modifications.